MantidQt/CustomInterfaces/src/Indirect/ContainerSubtraction.cpp

```diff
@@ -453,12 +441,41 @@ void ContainerSubtraction::absCorComplete(bool error) {
         "LogText", boost::lexical_cast<std::string>(m_uiForm.spShift->value()));
     m_batchAlgoRunner->addAlgorithm(shiftLog);
   }
+  // Enable post process plotting and saving
+  m_uiForm.cbPlotOutput->setEnabled(true);
+  m_uiForm.pbPlot->setEnabled(true);
+  m_uiForm.pbSave->setEnabled(true);
 
   // Run algorithm queue
   connect(m_batchAlgoRunner, SIGNAL(batchComplete(bool)), this,
           SLOT(postProcessComplete(bool)));
   m_batchAlgoRunner->executeBatchAsync();
 }
 
+/**
+ * Handles saving of workspace
+ */
+void ContainerSubtraction::saveClicked() {
+
+  // Check workspace exists
+  IndirectTab::checkADSForPlotSaveWorkspace(m_pythonExportWsName, false);
+  addSaveWorkspaceToQueue(QString::fromStdString(m_pythonExportWsName));
+}
```

Comment:
I have a question about the intended behaviour of `Save Results`: from the issue description I understand that when a user has run the reduction, they should be able to save the result just by clicking on `Save Result`. However, the result is only saved when `Run` is clicked again. The reason is that in the `saveClicked()` slot above, we are just adding the instruction to save the workspace to the list of algorithms to run, but those will not be run until the program reaches the first `m_batchAlgoRunner->executeBatchAsync()` which happens in line L176 in `ContainerSubtraction.cpp`.